operator/v1/types_machineconfiguration.go

```diff
@@ -494,3 +494,19 @@ const (
 	// Special represents an action that is internal to the MCO, and is not allowed in user defined NodeDisruption policies.
 	SpecialStatusAction NodeDisruptionPolicyStatusActionType = "Special"
 )
+
+// These strings will be used to indicate success and failures for MachineConfiguration knobs via conditions.
+// Please note: These are subject to change. Both additions and deletions may be made.
+const (
+	// MachineConfigurationBootImageUpdateFailed means that the MCO ran into an error while reconciling boot images.
+	// +openshift:enable:FeatureGate=ManagedBootImages
```

Comment:
this tag has no effect on this.  The way conditions work, we dont' restrict the condition types.